src/ontology/mondo-edit.obo

```diff
@@ -513701,6 +513701,17 @@ relationship: predisposes_towards MONDO:0013792 {source="OMIM:614519"} ! intrace
 property_value: http://purl.org/dc/terms/creator https://orcid.org/0000-0001-5208-3432
 property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/7038" xsd:anyURI
 
+[Term]
+id: MONDO:0100534
+name: SMARCB1-deficient kidney medullary carcinoma
+def: "A high-grade carcinoma that arises from the renal medulla and is characterized by inactivation of the SMARCB1 gene. It affects children and adults and occurs mainly in patients with sickle cell trait. The majority of the cases occur in the right kidney." [NCIT:C189247]
+xref: NCIT:C189247 {source="MONDO:equivalentTo"}
+is_a: MONDO:0006260 {source="NCIT:C189247"} ! kidney medullary carcinoma
+relationship: excluded_from_qc_check http://purl.obolibrary.org/obo/mondo/sparql/qc/general/qc-single-child.sparql
+relationship: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/11103 {source="NCIT:C189247"} ! SMARCB1
```

Comment:
@nicolevasilevsky Maybe this requires more discussions, but I am not sure that this axiom is correct. 
The definition indicates that the SMARCB1 gene is inactivated, but it could be because other reasons than "variation in germline".

Reply:
good point, I remvoved this